rollup.config.js

```diff
@@ -119,6 +124,7 @@ const componentList = {
     
     
 export default {
+    format: "iife",
```

Comment:
Why an IIFE here?

Reply:
Was following multiple guides/docs on getting babel working. I can remove this and see if it causes a problem.

Reply:
I’m curious why to transpire this down into an IIFE and how that impacts chunking. You should be able to see the output if you run the build script locally.